sounds.json

```diff
@@ -670,7 +670,9 @@
     "playsound_mapping": "mob.bogged.hurt",
     "bedrock_mapping": ""
   },
-  "entity.bogged.shear": {},
+  "entity.bogged.shear": {
+    "playsound_mapping": "mob.sheep.shear"
+  },
```

Comment:
are the sounds for sheep/bogged the same?

Reply:
Yes, here is the sound from the Java 1.21 sounds.json.
```json
 "entity.bogged.shear": {
    "sounds": [
      "mob/sheep/shear"
    ],
    "subtitle": "subtitles.item.shears.shear"
  },
  ```